libs/designsystem/modal/src/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -396,6 +398,19 @@ export class ModalWrapperComponent
     });
   }
 
+  private observeTitleContentChanges() {
+    observeContent(this.ionTitleElement.nativeElement, this.setTitleAsAriaLabel, this.willClose$);
+  }
+
+  private setTitleAsAriaLabel = () => {
+    if (!this.ionTitleElement) return;
+    const titleTextContent = this.ionTitleElement.nativeElement.textContent;
```

Comment:
Consider readability by adding a linebreak after the guard:
```suggestion
    if (!this.ionTitleElement) return;
    
    const titleTextContent = this.ionTitleElement.nativeElement.textContent;
```